deployment/deployment.go

```diff
@@ -0,0 +1,85 @@
+package deployment
+
+import (
+	"os"
+
+	"github.com/akitasoftware/akita-cli/printer"
+	"github.com/akitasoftware/akita-libs/tags"
+)
+
+type Deployment string
+
+const (
+	None       Deployment = ""
+	Unknown               = "unknown"
+	Kubernetes            = "kubernetes"
```

Comment:
I think Go wants us to specify `Deployment` on all declarations here; otherwise, only the first one is typed as `Deployment`, and the others are just untyped strings. Alas, the typing magic of `iota` doesn't apply here.

Reply:
Hm, I'm not sure why it compiled, but you're correct.